kafka-merge-pr.py

```diff
@@ -130,7 +130,15 @@ def merge_pr(pr_num, target_ref, title, body, pr_repo_desc):
                              '--pretty=format:%an <%ae>']).split("\n")
     distinct_authors = sorted(set(commit_authors),
                               key=lambda x: commit_authors.count(x), reverse=True)
-    primary_author = distinct_authors[0]
+    primary_author = raw_input(
+        "Enter primary author in the format of \"name <email>\" [%s]: " %
+        distinct_authors[0])
+    if primary_author == "":
+        primary_author = distinct_authors[0]
+
+    reviewers = raw_input(
+        "Enter reviewers in the format of \"name1 <email1>\", \"name2 <email2>\: ").strip()
```

Comment:
super nit: the string is slightly malformatted.  The script displays:
Enter reviewers in the format of "name1 <email1>", "name2 <email2>\:

Also, Its not clear if I should actually type the quotes when entering reviewers.